src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py

```diff
@@ -1289,8 +1289,7 @@ def download_from_original_stable_diffusion_ckpt(
     if controlnet is None:
         controlnet = "control_stage_config" in original_config.model.params
 
-    if controlnet:
-        controlnet_model = convert_controlnet_checkpoint(
+        controlnet = convert_controlnet_checkpoint(
             checkpoint, original_config, checkpoint_path, image_size, upcast_attention, extract_ema
         )
```

Comment:
https://github.com/iamwavecut/diffusers/pull/1 this might have introduced a bug

Reply:
It did indeed haha. Solved as explained in https://github.com/iamwavecut/diffusers/pull/1